files/image_config/interfaces/interfaces-config.sh

```diff
@@ -34,6 +34,48 @@ for intf_pid in $(ls -1 /var/run/dhclient*.Ethernet*.pid 2> /dev/null); do
     [ -f ${intf_pid} ] && kill `cat ${intf_pid}` && rm -f ${intf_pid}
 done
 
+
+# Setup eth1 if we connect to a remote chassis DB.
+PLATFORM=${PLATFORM:-`sonic-cfggen -H -v DEVICE_METADATA.localhost.platform`}
+PLATFORM_ENV="/usr/share/sonic/device/$PLATFORM/platform_env.conf"
+[ -f $PLATFORM_ENV ] && source $PLATFORM_ENV
+
+ASIC_CONF="/usr/share/sonic/device/$PLATFORM/asic.conf"
+[ -f $ASIC_CONF ] && source $ASIC_CONF
+
+if [[ -n "$MIDPLANE_SUBNET" && ($NUM_ASIC -gt 1) ]]; then
```

Comment:
The MIDPLANE_SUBNET should match with the "chassisdb_address" in the platform file eg: device/arista/x86_64-arista_7800_sup/chassisdb.conf ?

Reply:
Done.

Reply:
Since platform_env.conf and chassisdb.conf  are in the same format, also the subnet and db_address are related,  I will suggest to consolidate them into one file.  If we don't like the same of the file, we can rename it sometimes in the future.

Reply:
> Since platform_env.conf and chassisdb.conf are in the same format, also the subnet and db_address are related, I will suggest to consolidate them into one file. If we don't like the same of the file, we can rename it sometimes in the future.

I agree. A single generic file should be used for all chassis-specific configurations. There would be more configs in future, like http-config, and I suggest using a more scalable solution like a yaml config file.

Reply:
I moved this definition to `chassisdb.conf` and renamed it `midplane_subnet` (small caps) to be consistent with `chassisdb_address`.